### DISCUSSION TITLE
Waiting for Wormhole Network consensus

### DISCUSSION THREAD

**Discussion by Pygowsky** [wormhole-foundation/wormhole#2509] [Problems / bugs] [answered]
Hello, I submitted a conversion from USDC on Poly to ETH and am stuck at waiting for wormhole network consensus. When I view the transaction on Polygonscan it says success.

When I try to redeem the tokens I get the error "requested VAA not found in store"

Not sure what the best move would be next, thank you for any assistance!

EDIT: After about an hour it all went through. FYI for anyone else waiting and wondering.

**Comment by aadam-10** [comment, marked as answer]
@Pygowsky glad it's resolved now! In case you need help again, I suggest to try on Discord first as that way is usually faster. Find the Discord server link at the bottom of https://wormhole.com/ .
